src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -27,7 +30,16 @@ public LineResponse(Long id, String name, String color, List<StationResponse> st
     }
 
     public static LineResponse of(Line line, List<StationResponse> stations) {
-        return new LineResponse(line.getId(), line.getName(), line.getColor(), stations, line.getCreatedDate(), line.getModifiedDate());
+        return new LineResponse(line.getId(), line.getName(), line.getColor(), stations,
+                line.getCreatedDate(), line.getModifiedDate());
+    }
+
+    public static List<LineResponse> toLineResponses(List<Line> lines) {
+        return lines.stream()
+                .map(line -> {
+                    return of(line, StationResponse.toStationResponses(line.getSortedStations()));
+                })
```

Comment:
```suggestion
                .map(line -> of(line, StationResponse.toStationResponses(line.getSortedStations())))
```
이렇게 써볼수도 있겠네요!

또는 `LineResponse of(Line line)` 메서드를 정의하면
```suggestion
                .map(LineResponse::of)
```
이렇게 쓸수도 있겠네요 😀